020-LinuxFundamentals/Student/Challenge-06.md

```diff
@@ -0,0 +1,28 @@
+# Challenge 06 - Process management
+
+[< Previous Challenge](./Challenge-05.md) - **[Home](../README.md)** - [Next Challenge >](./Challenge-07.md)
+
+## Description
+
+In this challenge your objectives will involve basic process management, giving you real-world activities such as checking a list of processes running, identifying process ids' and discovering process execution in y our environment.
+
+- View the list of your server processes
+- How many processes are running on your server
```

Comment:
This line doesn't match the others in terms of subject verb agreement. Change to "Discover how many processes are running on your server"

Reply:
I think you didn't remove the original statement here?   I see "Discover how many processes are running on your server" with an entry under it saying "How many processes are running on your server"

The second line there should be removed.